lib/api/v3/work_packages/form/work_package_payload_representer.rb

```diff
@@ -76,6 +76,26 @@ def initialize(represented, options = {})
 
           property :lock_version
           property :subject, render_nil: true
+          property :done_ratio,
+                   as: :percentageDone,
+                   getter: -> (*) { done_ratio if Setting.work_package_done_ratio != 'disabled' },
+                   render_nil: false
```

Comment:
The setter is implicit here. Would you like one to make it appear more consistently?